ci-operator/step-registry/openshift/e2e/vsphere/csi/openshift-e2e-vsphere-csi-workflow.yaml

```diff
@@ -4,7 +4,6 @@ workflow:
     pre:
     - chain: ipi-vsphere-pre
     - ref: storage-conf-csi-vsphere
-    - ref: storage-conf-feature-gate
     - ref: storage-conf-wait-for-csi-driver
```

Comment:
storage-conf-wait-for-csi-driver can be removed too.

Reply:
done